docroot/modules/custom/foia_users/foia_users.module

```diff
@@ -92,11 +108,10 @@ function _form_validation_foiaoffice($form, $form_state) {
     }
 
     if (is_null($field_foia_office) && ($user_role_match)) {
-      $form['field_foia_office']['widget']['#required'] = TRUE;
-      $form_state->setErrorByName('field_foia_office', t('The roles "FOIA Office creator" and "FOIA Office reviewer" must be assigned to an office.'));
+      $form['field_agency']['widget']['#required'] = TRUE;
+      $form_state->setErrorByName('field_agency', t('The roles "Agency Component creator" and "Agency Component reviewer" must be assigned to an office.'));
```

Comment:
So is the code here essentially attempting to force someone who's editing this user to fill out the agency field if they have certain roles? If so, we should probably be making sure they have the new 'Agency Manager' role, and not these other roles.

Reply:
Interesting. I simply modified code that was already there to work with `field_agency` rather than `field_foia_office`.